src/test/functional/pages/BasePage.ts

```diff
@@ -25,10 +25,7 @@ export class BasePage {
     await this.page.waitForURL(config.baseUrl);
   }
 
```

Comment:
Please consider adding universal method for textBoxes in this project
```suggestion
protected getTextboxByTextLabel(label: string): Locator {
    return this.page.getByText(label, { exact: true})
        .locator("xpath=../..")
        .getByRole("textbox");
}
```